OpenGL/Constructs/VBO.cs

```diff
@@ -66,6 +68,11 @@ public class VBO<T> : IDisposable
             typeof(int),
             typeof(uint),
         };
+
```

Comment:
I don't suggest doing this, as it will duplicate your GPU data (which often does not need to be available to the CPU to use, causing memory overhead).  If you do need to maintain a copy of this data then I would suggest creating your own class based on VBO with the methods you have added.  I think it's worth exploring adding the `this[int row, int colt]` to `Matrix4`, but I don't think we should modify VBO to keep a copy of the data.  I suggest modifying this PR to remove changes to VBO.vs and move forward with changes to Matrix4.cs and Vector4.vs.

It could be worth exploring adding a VBO type to the project that does store this data, but I think it would be worth exploring in a separate PR and not forcing it for all VBO usage.